src/basic/main.basic.js

```
@@ -844,22 +742,22 @@ cartDisplay.addEventListener('click', function (event) {
         qtyElem.textContent = newQty;
-        prod.q -= qtyChange;
+        prod.quantity -= qtyChange;
       } else if (newQty <= 0) {
-        prod.q += currentQty;
+        prod.quantity += currentQty;
         itemElem.remove();
       } else {
         alert('재고가 부족합니다.');
       }
     } else if (tgt.classList.contains('remove-item')) {
       var qtyElem = itemElem.querySelector('.quantity-number');
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)